src/arrayMethodJoin.js

```diff
@@ -4,8 +4,29 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let string = '';
+    let count = 0;
+
+    this.forEach(el => {
+      if (this.indexOf(el) === 0 && count === 0) {
```

Comment:
```suggestion
    this.forEach((el, index) => {
      if (index === 0) {
```
У forEach есть index его можно использовать для проверки на нолевой елемент.